src/main/java/easytests/api/v1/controllers/UsersController.java

```diff
@@ -29,9 +35,24 @@
     /**
      * list
      */
-    /**
-     * create
-     */
+    @PostMapping("")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Identity create(@RequestBody User user) throws BadRequestException {
+        if (user.getId() != null) {
+            throw new IdentifiedModelException();
+        }
+        if (this.usersService.findByEmail(user.getEmail()) != null) {
+            throw new BadRequestException();
```

Comment:
I think that we should add some additional comments in that case, that e-mail already present